tutorials/dataframe/df003_profiles.py

```diff
@@ -12,12 +12,11 @@
 ## \author Danilo Piparo
 
 import ROOT
-RDataFrame = ROOT.ROOT.RDataFrame
 
 # A simple helper function to fill a test tree: this makes the example
 # stand-alone.
 def fill_tree(treeName, fileName):
-    d = RDataFrame(25000)
+    d = ROOT.ROOT.RDataFrame(25000)
```

Comment:
What is the reason to re-add this duplication? Is this not working properly in Py3? @dpiparo

Reply:
There is no problem running the tutorials with python 3 from the command line.

The issue has to do with scoping of variables when the tutorial is run from within the doxygen filter. The global scope is then the makeimage.py script and not the tutorial. So variables declared as "global" in the tutorial are no longer global when run inside the makeimage wrapper, and therefore not accessible in a local scope (like inside a function definition).

In this case the "RDataFrame = ROOT.ROOT.RDataFrame" earlier in the file is not visible inside the scope of fill_tree().